src/background_thread.c

```diff
@@ -285,6 +171,10 @@ static inline void
 background_work_sleep_once(tsdn_t *tsdn, background_thread_info_t *info, unsigned ind) {
 	uint64_t min_interval = BACKGROUND_THREAD_INDEFINITE_SLEEP;
 	unsigned narenas = narenas_total_get();
+	const purge_interval_opts_t purge_interval_opts = {
```

Comment:
Doesn't matter if we're making these just raw arguments instead of an options struct (as suggested above), but just as a note: we can't use designated initializers in jemalloc, because of limited MSVC support.